app/code/Magento/Pwa/Test/Mftf/Test/RegisteredUserCheckoutWithMultipleProducts.xml

```diff
@@ -8,24 +8,26 @@
 
 <tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
-    <test name="GuestCheckoutWithMultipleProducts" extends="CreateConfigurableProduct">
+    <test name="RegisteredUserCheckoutWithMultipleProducts" extends="CreateConfigurableProduct">
         <annotations>
             <features value="PWA"/>
-            <stories value="Guest Checkout"/>
-            <title value="Verify guest user should be able to checkout with Multiple products."/>
-            <description value="Verify guest user should be able to checkout with Multiple products."/>
+            <stories value="Checkout"/>
+            <title value="Verify registered user should be able to checkout with Multiple products."/>
+            <description value="Verify registered user should be able to checkout with Multiple products."/>
             <severity value="CRITICAL"/>
-            <testCaseId value="MC-15638"/>
+            <testCaseId value="MC-15720"/>
             <group value="PWA"/>
             <skip>
```

Comment:
Should we unskip now and assume magento-research/pwa-studio#968 will be merged together?

Reply:
I did unskip now as 1064 is closed. I will merge this with #968.